src/files-regular/add.js

```diff
@@ -125,7 +138,7 @@ module.exports = (createCommon, options) => {
 
       ipfs.add(data, (err) => {
         expect(err).to.exist()
-        expect(err.message).to.contain('invalid input')
+        expect(err.message).to.contain('Input not supported')
```

Comment:
Does this pass when these tests are run against `js-ipfs-http-client`?

Reply:
yes